risc0/bigint2/src/ec/mod.rs

```diff
@@ -16,124 +16,240 @@
 mod tests;
 
 use include_bytes_aligned::include_bytes_aligned;
-use num_bigint::BigUint;
 
-use crate::ffi::{sys_bigint2_2, sys_bigint2_3};
+use crate::ffi::{sys_bigint2_3, sys_bigint2_4};
 
 const ADD_BLOB: &[u8] = include_bytes_aligned!(4, "add.blob");
 const DOUBLE_BLOB: &[u8] = include_bytes_aligned!(4, "double.blob");
 
+/// The secp256k1 curve's prime as u32 digits, least significant digit first
+const SECP256K1_PRIME: [u32; 8] = [
+    0xFFFFFC2F, 0xFFFFFFFE, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF,
+];
+const SECP256K1_CURVE: &WeierstrassCurve<8> =
+    &WeierstrassCurve::<8>::new(SECP256K1_PRIME, [0u32; 8], [7, 0, 0, 0, 0, 0, 0, 0]);
+
 pub const EC_256_WIDTH_WORDS: usize = 256 / 32;
```

Comment:
Not currently used anywhere, I made some use in #2542 but we could also not expose this if we want